site/docs/components/divider/accessibility.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Note
+
```

Comment:
“Note” is used once in heading and once to start the sentence. I would say presentational rather than stylistic. I am not sure this is note is that clear, what are we trying to say ?
Maybe we add a note that these dividers are not designed to be focusable as then you would needed additional aria attributes ?
We could revisit this though, if we have a use-case. No need to complicate by trying to support things we don’t have a use-case for.

Reply:
modified